test/upgrade/functions.go

```diff
@@ -110,6 +114,7 @@ func handleStopEvent(
 	bc.Log.Infof("%s have received a stop event: %s", wc.Name, se.Name())
 	defer close(se.Finished)
 	wc.OnStop(se)
+	se.T.Log(bc.LogBuffer)
```

Comment:
Should this log every line on its own rather than the whole package in one line?

Additionally (because of the ordering "issue") should we log "Background logs:" and "Background logs end" explicitly, to clearly denote which logs are coming from the background process?